### PR TITLE
II-45: Changes capitalization to 'Identity Anchor'

### PR DESCRIPTION
**Pull request by kritzcreek**

### CODE REVIEW

File: src/frontend/src/flows/loginUnknown.ts [hunk 1]
```diff
@@ -39,19 +39,19 @@ const pageContent = (userIntent: UserIntent) => html` <style>
...
       id="registerUserNumber"
-      placeholder="Enter Identity anchor"
+      placeholder="Enter Identity Anchor"
     />
     <button type="button" id="loginButton" class="primary">Authenticate</button>
     ${userIntent.kind === "addDevice"
       ? html`<div class="spacer"></div>`
       : html`<div class="textLink" id="registerSection">
-            New user?
+            New?
```
Comment (robin-kunzler) [inline src/frontend/src/flows/loginUnknown.ts:54]: Just because this seems to be an unrelated change: Do you actually want to change "New user" to "New"?

**Review by robin-kunzler — APPROVED** [review]
@kritzcreek : Thanks for adapting the capitalization! Looks good to me. 

* I also did a search through the code base and did not find any other mentions of identity anchor where the capitalization is still wrong. 
* Only have one minor question, see below.

File: src/frontend/src/flows/loginUnknown.ts (hunk 1 above)
Comment (kritzcreek) [reply src/frontend/src/flows/loginUnknown.ts:54]: Good spot. Jan asked me to make that change so I added it but it doesn't really fit the topic of this PR. Would you prefer I move it to a later terminology PR?

Comment (robin-kunzler) [reply src/frontend/src/flows/loginUnknown.ts:54]: I'm completely fine with leaving it in this PR, just asked to make sure that this change was intentional!

Comment (kritzcreek) [reply src/frontend/src/flows/loginUnknown.ts:54]: Great! Understood.
